proc_fusion_spell.lua

```diff
@@ -255,20 +284,99 @@ function (fusfilter,matfilter,extrafil,extraop,gc2,stage2,exactcount,value,locat
 					if sel[1]==e then
 						Fusion.CheckAdditional=checkAddition
 						Fusion.ExtraGroup=extragroup
+						--"Cyberdark Chimera" OPT EFFECT_EXTRA_FUSION_MATERIAL check
+						--Same checks etc as line 119.
+						local extra_feff_mg=mg1:Filter(Card.IsHasEffect,nil,EFFECT_EXTRA_FUSION_MATERIAL)
+						if #extra_feff_mg>0 then
+							local extra_feff=extra_feff_mg:GetFirst():IsHasEffect(EFFECT_EXTRA_FUSION_MATERIAL)
+							if extra_feff and Duel.GetFlagEffect(tp,extra_feff:GetHandler():GetCode())>0 then
+								if extrafil then
+									local extrafil_g=extrafil(e,tp,mg1)
+									if #extrafil_g>0 and not extrafil_g:IsExists(Card.IsLocation,1,nil,LOCATION_GRAVE) then
+										mg1:Sub(extra_feff_mg:Filter(Card.IsLocation,nil,LOCATION_GRAVE))
+									end
+								else
+									mg1:Sub(extra_feff_mg:Filter(Card.IsLocation,nil,LOCATION_GRAVE))
+								end
+							end
+						end
 						local mat1=Duel.SelectFusionMaterial(tp,tc,mg1,gc,chkf)
 						Fusion.ExtraGroup=nil
 						backupmat=mat1:Clone()
 						tc:SetMaterial(mat1)
+						--Checks for the case that the Fusion effect has an "extraop"
+						extra_feff_mg=mat1:Filter(Card.IsHasEffect,nil,EFFECT_EXTRA_FUSION_MATERIAL)
+						if #extra_feff_mg>0 and extraop then
```

Comment:
Shouldn't `extra_feff_mg` remain the same from the previous call?

Reply:
No.